pom.xml

```diff
@@ -50,12 +50,8 @@
           <artifactId>error_prone_annotations</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>com.google.j2objc</groupId>
-          <artifactId>j2objc-annotations</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>animal-sniffer-annotations</artifactId>
+          <groupId>com.google.guava</groupId>
+          <artifactId>listenablefuture</artifactId>
```

Comment:
Optional dependency since 27.0, not needed since ListenableFuture is already part of guava. Note: this jar has no sources so would trip up maven-shade-plugin:3.0.0 if it were included (https://issues.apache.org/jira/browse/MSHADE-247)